deploy.sh

```diff
@@ -17,7 +17,7 @@ echo "ENV: $env"
 echo "Region: $region"
 
 # Download model artifacts
-dvc pull
+dvc pull -R models
 
 # Build container and deploy functions
-serverless deploy --stage "${env}" --region "${region}"
+npx serverless deploy --stage "${env}" --region "${region}"
```

Comment:
fixing stale script problem (do we want to use `deploy.sh` in the automatic deploys too?)

Reply:
Yeah, we should do it the same way in our deploy that we expect other people to use

Reply:
@karlhigley just pushed a new change that updates the deploy workflow to do this.